akka-remote/src/main/scala/akka/remote/serialization/PrimitiveSerializers.scala

```diff
@@ -112,7 +112,7 @@ class ByteStringSerializer(val system: ExtendedActorSystem) extends BaseSerializ
   }
 
   override def fromBinary(bytes: Array[Byte], manifest: Option[Class[_]]): AnyRef = {
-    ByteString(bytes)
+    ByteString.fromArrayUnsafe(bytes)
```

Comment:
I'm pretty sure this is not a buffer, but an array allocated for the call, right?